### DISCUSSION TITLE
Only one file cannot show bookmark

### DISCUSSION THREAD

**Discussion by ALPNZ** [sumatrapdfreader/sumatrapdf#2962] [Q&A]
A pdf file cannot show bookmark in sumatrapdf but it can show bookmark in foxit and adobe acrobat. The file could show bookmark before. But I do not what I do, suddenly it cannot show bookmark. The button of "show bookmark" is grey.
Other files can still display bookmark.

**Comment by GitHubRulesOK** [comment]
cant know without seeing file, 
but if bookmarks were available in same file in the past but not now then somehow they were removed or corrupted, or you found a file related version bug but no way to check without sample.

**Reply by ALPNZ** [reply]
I solved it after I imported bookmarks again.
Now it shows bookmarks.
Thanks for your replying.
